docs/03-computation/02-configuration.mdx

```diff
@@ -8,38 +8,49 @@ import Legislation from '../_partials/_legislation.mdx';
+<Legislation>
+This API allows you to define what is usually called <a href="https://www.cnil.fr/en/record-processing-activities" target="_blank">RoPA - Record of Processing Activities</a>, a way to comply with the requirements of Article 30 of GDPR but also with various CCPA requirements.
+</Legislation>
 
-- [General information](#configuring-general-information) about your Organization and its' data processing policies (Equivalent to a [RoPA - Record of Processing Activities](https://www.cnil.fr/en/record-processing-activities))
-- Mappings of your data structures to [PRIV](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md) Data Categories
-- Few choices you have to make about when to authenticate the user (before or after they make a Privacy Request), and when to deliver automatic responses and when to wait for validation
+In blindnet DevKit defining a RoPA means configuring the following:
+- [General information](#general-information) about your Organization, its representatives, its server's locations and policies
+- [Selectors](#selectors) - additional [PRIV](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md) Data Categories, subcategories of the existing ones, to describe the data you are processing.
+- [Legal Bases](#legal-bases) for processing of particular Data Categories for Particular Purposes
+- [Retention Policies](#retention-policies) specifying minimal and maximal retention times for particular Data Categories
+- [Provenances](#provenances) defining where your system gets the data from
+
+In addition, you can configure how you want the PCE to behave:
+- [Demand Resolution Strategy](#demand-resolution-strategy) : when (for what kind of Privacy Requests) to deliver automatic responses and when to wait for human validation
+- [Regulations](#regulations): add additional rule sets specific to certain regulations
 
-## General Information
 
 :::tip API
 
-General information can be set in the PCE using [PUT /configure/general-info](https://stage.computing.blindnet.io/swagger/#/Configuration/putV0ConfigureGeneral-info), retrieved using [GET /configure/general-info](https://stage.computing.blindnet.io/swagger/#/Configuration/getV0ConfigureGeneral-info), as detailed in the related `GeneralInformation` schema.
+In order to follow through this section of the tutorial, you might want to either [set-up your own instance of PCE](/docs/tutorials/intro/PCE-set-up) or use the PCE isntance provided by blindnet for testing at https://devkit-pce-staging.azurewebsites.net/v0/. In the followign examples we'll use this instance.
 
-:::
+If you are using the instance provided by blindnet you will need to reach out to us to get your `app_id` and `app_key`. Once you have got those, you can use our [Token Generator](https://blindnet-io.github.io/demo-token-generator/) to generate the credentials that you will need in order to authenticate your calls to our APIs.
```

Comment:
Provide a way to reach out.

Reply:
It can be the [contact page ](https://www.blindnet.io/contact). I thaught you were working on this. If not, and if you need me to do it, feel free let me know.